editor/plugins/org.fusesource.ide.project/src/org/fusesource/ide/project/camel/CamelFacetInstallationDelegate.java

```diff
@@ -195,8 +195,8 @@ private void createUtilityManifest(IProject project, IFolder metainf, IProgressM
 		
 		sb.append("Manifest-Version: 1.0\n");
 		sb.append("Bundle-ManifestVersion: 2\n");
-		sb.append("Bundle-Name: " + project.getName() + "\n");
-		sb.append("Bundle-SymbolicName: " + project.getName() +"\n"); // TODO this string won't be osgi-valid as a symbolic name possibly
+		sb.append("Bundle-Name: " + project.getName().getBytes(StandardCharsets.UTF_8) + "\n");
+		sb.append("Bundle-SymbolicName: " + project.getName().getBytes(StandardCharsets.UTF_8) +"\n"); // TODO this string won't be osgi-valid as a symbolic name possibly
 		sb.append("Bundle-Version: 1.0.0.SNAPSHOT\n");
 		if( manifestFile.exists())
 			manifestFile.setContents(new ByteArrayInputStream(sb.toString().getBytes()), IResource.FORCE, monitor);
```

Comment:
hum the StandardCharsets.UTF_8 was for new ByteArrayInputStream(sb.toString().getBytes())
I think that we don't need ti onporject.getName() directly